test/integration/sequelize.test.js

```diff
@@ -32,13 +32,13 @@ var qq = function(str) {
 describe(Support.getTestDialectTeaser('Sequelize'), function() {
   describe('constructor', function() {
     if (dialect !== 'sqlite') {
-      it.skip('should work with minConnections', function() {
+      it.skip('should work with min connections', function() {
```

Comment:
I was just wondering why it doesn't work, and if it may be fixed now - But I'll merge it as is ;)